scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -0,0 +1,410 @@
+version 1.0
+
+workflow GvsCalculatePrecisionAndSensitivity {
+  input {
+    Array[File] input_vcfs
+    String output_basename
+
+    String chromosome = "chr20"
+
+    Array[String] sample_names
+    Array[File] truth_vcfs
+    Array[File] truth_vcf_indices
+    Array[File] truth_beds
+
+    File ref_fasta
+  }
+
+  if (false) {
+    String? none = "None"
+  }
+
+  String? contig = if (chromosome == "all") then none else chromosome
+
```

Comment:
nit---I found this bit (lines 18-22) a lil confusing without a comment. And why are we passing the contig to SelectVariants if we already split by contig??? hmm